getting_started/create_vm/make_vm.sh

```diff
@@ -2,19 +2,25 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the Apache 2.0 License.
 
-set -ex
+set -e
+
+if [ "$#" -lt 1 ]; then
+  echo "make_vm.sh takes at least an Azure subscription name,"
```

Comment:
As above, would prefer `"Usage: $0 SUBSCRIPTION_NAME [path/to/ssh/public/key]"`

Reply:
Yes

Reply:
fixed